lucien-ios/AddNewComicBook/AddComicViewController.swift

```diff
@@ -311,4 +315,35 @@ final class AddComicViewController: UIViewController, UIPickerViewDelegate, UIPi
             }
         }
     }
+
+    // MARK: - CameraViewDelegate
+    
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Trailing Whitespace Violation: Lines should not have trailing whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Atrailing_whitespace)